src/test/java/uk/gov/hmcts/reform/sandl/snlapi/security/CustomUserDetailsServiceTest.java

```diff
@@ -0,0 +1,36 @@
+package uk.gov.hmcts.reform.sandl.snlapi.security;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.security.core.userdetails.UsernameNotFoundException;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@RunWith(SpringRunner.class)
+public class CustomUserDetailsServiceTest {
+
+    CustomUserDetailsService cuds;
+
+    public CustomUserDetailsServiceTest() {
+        this.cuds = new CustomUserDetailsService();
+    }
+
+    @Test
+    public void loadUserByUsername_whenUserExistsItReturnsItsUserDetails() {
+        String username = "officer1";
+
+        UserDetails officer1 = cuds.loadUserByUsername(username);
+
+        assertThat(officer1.getUsername()).isEqualTo(username);
+    }
+
+    @Test(expected = UsernameNotFoundException.class)
+    public void loadUserByUsername_whenUserDoesNotExistsItThrowsException() {
```

Comment:
If you don't catch the content then it's fine. (the missing Kamil Z comment)
If you would like to check if the message is correct then, indeed a double assert, on in try catch and one afterwards.